website/mails/mails.py

```diff
@@ -435,6 +435,11 @@ def get_english_article(word):
     subject='There was an error creating a DOI for preprint(s). batch_id: ${batch_id}'
 )
 
+CROSSREF_DOIS_PENDING = Mail(
+    'crossref_doi_pending',
+    subject='There are ${pending_doi_count} pending.'
```

Comment:
Could you make the subject of this email a little more clear? nothing will be sent with the word crossref in it, it will just say `There are x pending`.